view/rooms/new/app.js

```diff
@@ -34,7 +34,7 @@ function getPathVariables(adr){
   }
 
   const paramType = qdata.roomType || '';
-  const paramActive = qdata.active === 'true';
+  const paramActive = qdata.active !== 'false';
```

Comment:
so the intended behavior here is for any value other than the string "false" to be treated as true?

Reply:
yeah, default everything to active unless specifically false

Reply:
I wouldn't object to a comment on this line saying just that ;)